format-specs/geoparquet.md

```diff
@@ -1,5 +1,51 @@
+# Geospatial Parquet format
 
-## A Geospatial Parquet format
+## Overview
 
-This page is a scratch pad for ideas to define a 'geo-parquet' format, aligning on a default schema and format for geospatial data in Parquet.
+The [Apache Parquet][parquet] provides a standardized open-source columnar storage format. This specification defines how geospatial data
+should be stored in parquet format, including the representation of geometries and the required additional metadata.
 
+## Geometry columns
+
+Geometry columns are stored using the `BYTE_ARRAY` parquet type. They are encoded as [WKB](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Well-known_binary).
+
+## Metadata
+
+geoparquet files include additional metadata at two levels:
+
+1. File metadata indicating things like the version of this specification used
+2. Column (chunk) metadata with additional metadata for each geometry column
+
+## File metadata
+
+All file-level metadata should be included under the "geoparquet" key in the parquet metadata.
```

Comment:
This is new since your last review @cholmes. The motivation is to namespace things under "geoparquet" to avoid clashing with other systems using parquet metadata.